config/transactions/testTransferOwnershipVault.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we name this transaction something more specific in case there are more vault transfers in the future? Not blocking though.